raiden/tests/integration/cli/test_cli.py

```diff
@@ -0,0 +1,155 @@
+import json
+import pytest
+import pexpect
+import sys
+from click.testing import CliRunner
+from raiden.ui.cli import run
+from eth_utils import to_checksum_address
+
+
+def test_cli_version():
+    runner = CliRunner()
+    result = runner.invoke(run, ["version"])
+    assert result.exit_code == 0
+    result_json = json.loads(result.output)
+    result_expected_keys = ["raiden", "python_implementation", "python_version", "system"]
+    for expected_key in result_expected_keys:
+        assert expected_key in result_json
+
+
+@pytest.mark.timeout(25)
+def test_cli_full_init(cli_args):
+    child = pexpect.spawn('raiden', cli_args, logfile=sys.stdout)
```

Comment:
This way of spawning raiden is a bit too fragile for my taste.
I'd suggest to use `pexpect.spawn(sys.executable, ['-m', 'raiden'] + args, ...)` instead. That way it's guaranteed that we'll use the same python that pytest is running under even in cases where the users PATH is screwed up and we also don't depend on people having run `pip install -e .` before running the tests.

Reply:
Very good idea. For the record, I had to run `python setup.py install` for the tests to work. And your suggestion most probably fixes that. Let me implement it.